### PR TITLE
Adjust liquid flat swapout fee

### PR DESCRIPTION
**Pull request by nepet**
We adjust the estimated swap-out tx size that is used for fee calculation to 3000 vBytes. The size is based on 3 inputs and 3 outputs of which 2 are blinded (2672 vBytes), together with a security margin.

### CODE REVIEW

File: cmd/peerswaplnd/peerswapd/main.go [hunk 1]
```diff
@@ -103,6 +104,9 @@ func run() error {
 	}
 	log.Infof("PeerSwap LND starting up with commit %s and cfg: %s", GitCommit, cfg)
 	log.Infof("DB version: %s, Protocol version: %d", version.GetCurrentVersion(), swap.PEERSWAP_PROTOCOL_VERSION)
+	if isdev.IsDev() {
+		log.Infof("Devmode eneabled.")
```
Comment (wtogami) [inline cmd/peerswaplnd/peerswapd/main.go:108]: spelling

**Review by wtogami — APPROVED** [review]
Looks fine but fix the spelling before merge

**Comment by grubles** [comment]
Tested L-BTC swap-outs from both LND v0.15.3-beta and CLN (e00857827fd5b45a3022753edf855df4765a4e93) signet nodes and haven't run into any issues.

File: cmd/peerswaplnd/peerswapd/main.go (hunk 1 above)
Comment (nepet) [reply cmd/peerswaplnd/peerswapd/main.go]: 12a372e

**Pull request merged**